recipes/datasets/plugins/plugin_consensus.py

```diff
@@ -0,0 +1,63 @@
+from kili.plugins import PluginCore
+from kili.types import Label
+
+
+class PluginHandler(PluginCore):
+    """
+    Custom plugin instance
+    """
+
+    def aggregate_object_detection_label(self, object_job_key, labels_array):
+
+        json_response = {}
+
+        for key in object_job_key:
+            job_annotation = []
+            for label in labels_array:
+                if label["jsonResponse"][key]:
+                    for annotation in label["jsonResponse"][key]["annotations"]:
+                        job_annotation.append(annotation)
+            json_response.update({key: {"annotations": job_annotation}})
+
+        return json_response
+
+    def on_submit(self, label: Label, asset_id: str) -> None:
+        """
+        Dedicated handler for Submit action
+        """
+        project_id = self.project_id
+
+        project_info = list(
+            self.kili.projects(project_id=project_id, fields=["minConsensusSize", "jsonInterface"])
+        )[0]
+
+        json_interface = project_info["jsonInterface"]
+        min_consensus_size = project_info["minConsensusSize"]
+
+        object_job_key = [
+            k for k, v in json_interface["jobs"].items() if v["mlTask"] == "OBJECT_DETECTION"
+        ]
+
+        labels_array = self.kili.labels(
+            project_id=project_id,
+            asset_id=asset_id,
+            type_in=["DEFAULT"],
+            fields=[
+                "id",
+                "jsonResponse",
+                "isLatestLabelForUser",
+            ],
+        )
+
+        labels_array_last = list({})
+        for label_iteration in labels_array:
```

Comment:
https://www.programiz.com/python-programming/methods/built-in/filter

latest_labels = filter(x: x["isLatestLabelForUser"] is True, labels_array)